python/api/main.py

```diff
@@ -167,9 +167,26 @@ async def get_hfp_points(stop_id: str):
                 )
 
             observation_geojson_features = []
-
             for observation_tuple in totalObservations:
                 observation = observation_tuple[0]
                 observation_geojson_features.append(observation)
 
     return get_feature_collection(observation_geojson_features)
+
+
+@app.get("/percentile_circles/{stop_id}")
+async def get_percentile_circles(stop_id: str):
+    """
+    Returns a GeoJSON FeatureCollection of percentile circles with given stop_id.
```

Comment:
Suggested: `Returns a GeoJSON FeatureCollection of percentile circles around the given stop by stop_id.`